spec/unit/puppet_x/racadm_spec.rb

```diff
@@ -4,14 +4,20 @@
 require 'puppet_x/racadm/racadm'
 describe Racadm::Racadm do
 
-  let(:getIDRAC) { IO.read(
+  let(:getIDRAC_IPv4) { IO.read(
       File.join(
           File.dirname(__FILE__),
           '..', '..', 'fixtures', 'bmc', 'racadm7_getIDRAC.IPv4.txt'))
   }
 
-  context 'racadm7 get IDRAC.IPv4' do
-    subject { Racadm::Racadm.parseiDRAC_IPv4 getIDRAC }
+  let(:getIDRAC_LDAP) { IO.read(
+      File.join(
+          File.dirname(__FILE__),
+          '..', '..', 'fixtures', 'bmc', 'racadm7_getIDRAC.LDAP.txt'))
+  }
+
+  describe 'racadm7 get IDRAC.IPv4' do
```

Comment:
use context here. context is an alias for describe, but the standard says to use it like:
describe ... do
  context ... do
  end
end